src/components/experiment-tracking/run-details-modal/run-details-modal.js

```diff
@@ -87,6 +87,7 @@ const RunDetailsModal = ({
             defaultValue={valuesToUpdate.title}
             onChange={(value) => onChange('title', value)}
             resetValueTrigger={visible}
+            type="textarea"
```

Comment:
hi, what is this for ?

Reply:
`<Input>` component uses <textarea> internally but its not suitable for input fields in sharable-viz modal so I modified `<Input>` component to accept both the type `textarea` & `input` 
See here [Input component](https://github.com/kedro-org/kedro-viz/pull/1718/files#diff-1264eb0cfb0afbaed97f98d83a95263a0e1434850cae86844952eaeef7caa24f)